components/UnsafeForm.tsx

```
@@ -0,0 +1,141 @@
+    console.log('SSN:', formData.ssn)
+    
+    // Storing sensitive data in localStorage - SECURITY ISSUE
+    localStorage.setItem('userCreditCard', formData.creditCard)
+    localStorage.setItem('userSSN', formData.ssn)
+    
+    // Sending sensitive data over GET request - SECURITY ISSUE
+    fetch(`/api/submit?email=${formData.email}&password=${formData.password}&cc=${formData.creditCard}`)
+      .then(response => response.json())
+      .then(data => {
+                 // Direct DOM manipulation - REACT ANTI-PATTERN
+         document.getElementById('result')!.innerHTML = `<p>Welcome ${data.name}!</p>`
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)